modules/administration/pages/prometheus.adoc

```diff
@@ -14,7 +14,7 @@ It is used to create dashboards with panels representing specific metrics over a
 Grafana is commonly used together with Prometheus, but also supports other data sources such as ElasticSearch, MySQL, PostgreSQL, and Influx DB.
```

Comment:
Since we are removing all the references to Grafana, should we keep this paragraph and the one above, where we present what Grafana is?

In case we still want to present Grafana here - although not shipping the package - maybe we could also include the link to the SUSE Manager dashboard on grafana.com (below, line ~216), since users that manually download and install Grafana might still benefit from this.

Reply:
with my 2nd commit this is back again